curriculum/js1/6.js

```diff
@@ -5,8 +5,10 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (num, i=2,sum=0) => {
+  if(i===num) { return sum};
```

Comment:
what is num is 1? Then your code might run forever. Base case should be `i >= num`

Reply:
fixed